### PR TITLE
Added page to user DTO. added new page on new user. Fixed user tests

### PR DESCRIPTION
**Pull request by RPHaynes**

### CODE REVIEW

**Review by AaronLitton — COMMENTED** [review]
Nice job fixing the test. Looks good.😃

**Review by AaronLitton — APPROVED** [review]

File: src/main/java/com/revature/RevRelay/services/UserService.java [hunk 1]
```diff
@@ -81,6 +85,16 @@ public User createUser(UserRegisterAuthRequest userAuthRequest) throws IllegalAr
             user.setEmail(userAuthRequest.getEmail());
             user.setUsername(userAuthRequest.getUsername());
             user.setPassword(passwordEncoder.encode(userAuthRequest.getPassword()));
+			user = userRepository.save(user);
+			Page p = new Page();
+			p.setPosts(null);
+			p.setBannerURL("");
+			p.setDescription("You discription here");
```
Comment (LutherNa) [inline src/main/java/com/revature/RevRelay/services/UserService.java:92]: Typo

File: src/main/java/com/revature/RevRelay/services/UserService.java [hunk 2]
```diff
@@ -81,6 +85,16 @@ public User createUser(UserRegisterAuthRequest userAuthRequest) throws IllegalAr
             user.setEmail(userAuthRequest.getEmail());
             user.setUsername(userAuthRequest.getUsername());
             user.setPassword(passwordEncoder.encode(userAuthRequest.getPassword()));
+			user = userRepository.save(user);
+			Page p = new Page();
```
Comment (LutherNa) [inline src/main/java/com/revature/RevRelay/services/UserService.java:89]: This should be a special constructor on Page, or at least a helper method.

**Review by LutherNa — CHANGES_REQUESTED** [review]
Please make a separate method for building a new empty user page, either as a special constructor on Page or as a helper method.

**Review by LutherNa — APPROVED** [review]
Thank your for your changes. Pull request APPROVED.

**Pull request merged**